docs/timezone.md

```diff
@@ -20,16 +20,16 @@ There are two ways to do this.
 The recommended way is to create a class inheriting from `Temporal.TimeZone`.
 You must use one of the built-in time zones as the "base time zone".
 In the class's constructor, call `super()` with the identifier of the base time zone.
-The class must override `toString()` to return its own identifier.
-Overriding `getOffsetNanosecondsFor()`, `getPossibleInstantsFor()`, `getNextTransition()`, and `getPreviousTransition()` is optional.
-If you don't override the optional members, then they will behave as in the base time zone.
-You don't need to override any other methods such as `getOffsetStringFor()` because they will call `getOffsetNanosecondsFor()`, `getPossibleInstantsFor()`, and `toString()` internally.
+The class must override the `id` property to be its own identifier.
```

Comment:
The docs for `id` also need to be updated, because they currently say this: 

> When subclassing `Temporal.TimeZone`, this property doesn't need to be overridden because the default implementation gives the result of calling `toString()`.

Reply:
Fixed this, for calendars as well.